src/bun.js/api/bun/process.zig

```diff
@@ -0,0 +1,793 @@
+const bun = @import("root").bun;
```

Comment:
later it would be nice to merge this file with spawn.zig.PosixSpawn, which usingnamespaces this file.

this file should be named spawn.zig and bun.zig should just do `const spawn = @import("spawn.zig")`. the current way that `bun.spawn` points to this file is much more convoluted than it needs to be.